src/features/moviesPage/moviePage/index.js

```diff
@@ -7,6 +7,8 @@ import Loader from "../../../common/Loader";
 import { imagesAPI600x900 } from "../../../core/API";
```

Comment:
once we have all the necessary content, we will adjust the size of the photos to what we really need